panther_power_control/src/power_board_node.py

```diff
@@ -52,26 +33,56 @@ def turn_on(self) -> None:
     def turn_off(self) -> None:
         self._enabled = False
         self._last_state = False
-        GPIO.output(self._pin, self._last_state)
+        self._watchdog_pin.set_value(self._last_state)
 
 
 class PowerBoardNode:
     def __init__(self, name: str) -> None:
-        rospy.init_node(name, anonymous=False)
+        self._node_name = name
+        rospy.init_node(self._node_name, anonymous=False)
 
         self._pins_lock = Lock()
         self._e_stop_lock = Lock()
 
         self._clearing_e_stop = False
-        self._pins = PatherGPIO()
 
-        self._setup_gpio()
+        out_line_names = [
+            'AUX_PW_EN',  # Enable auxiliary power, eg. supply to robotic arms etc.
+            'CHRG_DISABLE',  # Disable charger
+            'DRIVER_EN',  # Enable motor drivers (1 - on)
+            'FAN_SW',  # Turn on the fan (1 - on)
+            'VDIG_OFF',  # Turn the digital power off eg. NUC, Router etc. (1 - off)
+            'VMOT_ON',  # Enable mamin power supply to motors (1 - on)
+            'WATCHDOG',  # Watchdog pin, if PWM is on this pin Panther will work
+        ]
+        in_line_names = [
+            'CHRG_SENSE',  # Charger sensor (1 - charger plugged in)
```

Comment:
This is active low pin, which means 1 - charger not connected
Set this using LINE_REQ_FLAG_ACTIVE_LOW or make clear comment that this pin is in inverted logic